src/server/scripts/Northrend/Ulduar/Ulduar/boss_xt002.cpp

```diff
@@ -374,7 +375,7 @@ class boss_xt002 : public CreatureScript
                     heart->CastSpell(heart, SPELL_HEART_OVERLOAD, false);
                     heart->CastSpell(me, SPELL_HEART_LIGHTNING_TETHER, false);
                     heart->CastSpell(heart, SPELL_HEART_HEAL_TO_FULL, true);
-                    heart->CastSpell(heart, SPELL_EXPOSED_HEART, false);    // Channeled
+                    heart->AddAura(SPELL_EXPOSED_HEART, heart);    //Hack Fix that applies 2x damage aura
```

Comment:
Maybe because heart is "moving" when cast is called.
You can cleanup all "CastSpell" with "false" argumen,  triggered = false by default, so can be omitted